pkg/registry/extensions/rest/storage_extensions.go

```diff
@@ -70,7 +70,7 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 	storage["deployments/scale"] = deploymentStorage.Scale
 	// ingresses
 	ingressStorage, ingressStatusStorage := ingressstore.NewREST(restOptionsGetter)
-	storage["ingresses"] = ingressStorage
+	storage["ingresses"] = ingressStorage.WithCategories(nil)
```

Comment:
why are you overriding this to nil here? that has only been done in other places to prevent the same resource in multiple groups (deployments, replicasets, etc) from appearing twice in `kubectl get all`

Reply:
@kubernetes/sig-network-api-reviews what is the plan for moving ingresses out of the `extensions/v1beta1` API group? It is the only resource to not have a destination API group planned (all other resources have been relocated and their extensions/v1beta1 counterparts are deprecated). I'd like to see ingresses moved to their final home and *that* resource be added to the `all` category